README.md

```diff
@@ -13,80 +13,114 @@ You can find deb/rpm packages on the release page.
-
-Fedora  
-`sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext kf5-kcmutils-devel`  
-
-Arch(Manjaro):  
-`sudo pacman -S cmake extra-cmake-modules kcmutils`  
-
-### Build instructions
-
-The easiest way to install is:  
-`curl https://raw.githubusercontent.com/alex1701c/krunner-firefox/master/install.sh | bash`  
-Or you can install it without admin privileges:  
-`curl https://raw.githubusercontent.com/alex1701c/krunner-firefox/master/install-user.sh | bash`  
-
-Or you can do it manually:  
-```
-git clone https://github.com/alex1701c/krunner-firefox
-cd krunner-firefox
-mkdir build
-cd build
-cmake -DCMAKE_BUILD_TYPE=Release -DKDE_INSTALL_QTPLUGINDIR=$(kf5-config --qt-plugins) ..
-make
-sudo make install
-kquitapp5 krunner 2> /dev/null; kstart5 --windowclass krunner krunner > /dev/null 2>&1 &
-```
-
-To get the icon for the private window you have to extract it:   
-```
-mkdir -p /tmp/firefoxprofile_installer
-unzip /usr/lib/firefox/browser/omni.ja -c "chrome/browser/skin/classic/browser/privatebrowsing/favicon.svg" -d /tmp/firefoxprofile_installer > /dev/null 2>&1
-sudo mv /tmp/firefoxprofile_installer/chrome/browser/skin/classic/browser/privatebrowsing/favicon.svg /usr/share/pixmaps/private_browsing_firefox.svg
-rm -rf /tmp/firefoxprofile_installer
-```
+Debian/Ubuntu:
+
+    sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libkf5kcmutils-dev
+
+openSUSE:
+
+    sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel ktextwidgets-devel kservice-devel krunner-devel gettext-tools kconfigwidgets-devel kcmutils-devel  
+
+Fedora:
+
+    sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext kf5-kcmutils-devel  
+
+Arch/Manjaro:
+    sudo pacman -S cmake extra-cmake-modules kcmutils
+
+### Quick Install
+
+Download the latest install script:
+
+    curl https://raw.githubusercontent.com/alex1701c/krunner-firefox/master/install.sh
+
+To install as a global plugin:
+
+    bash install.sh
+
+To install as a user plugin, admin privileges not needed:
+
+    GLOBAL_PLUGIN=false bash ./install.sh
+
+The install script can also be executed from the cloned repository:
+
+    git clone https://github.com/alex1701c/krunner-firefox
+    cd krunner-firefox
+    GLOBAL_PLUGIN=true bash ./install.sh
+
+### Manual build and install instructions
+
+To install as a global plugin:
+
+    git clone https://github.com/alex1701c/krunner-firefox
+    cd krunner-firefox
+    mkdir build
+    cd build
+    cmake -DCMAKE_BUILD_TYPE=Release -DKDE_INSTALL_QTPLUGINDIR=$(kf5-config --qt-plugins) ..
+    make
+    sudo make install
```

Comment:
This command isn't just for the actual installation, here the make target generates a bunch of files for preparing the installation and you need to use `sudo` to delete the build directory after a system install.

I will look into this in the future, maybe we can avoid having files owned by root. I'm thinking that we could have: `make install-user` and `make install-system` and just use sudo for copying the files in a separate make target.

Reply:
You mean the `install_manifest.txt` file? Then the dir can still be deleted with the `-f` option.

But I see that you have just used `make`, I would prefer `make -j$(nproc)` to compile faster.

Reply:
Added the jobs option to the install script and in the README.md

To delete the build , you'll need to run `sudo rm -rf build`, since both the owner and the group will be for the root user.